bessctl/conf/samples/vxlan.bess

```diff
@@ -36,33 +36,34 @@ Container Bob  |    (regular Ethernet port)    |
 
 import os
 import socket
+import struct
 
-def aton(ip):
-    return socket.inet_aton(ip)
+def atoh(ip):
+    return struct.unpack("!L", socket.inet_aton(ip))[0]
 
-os.system('docker pull ubuntu > /dev/null')
-os.system('docker run -i -d --net=none --name=vxlan_test ubuntu /bin/bash > /dev/null')
+os.system('sudo docker pull ubuntu > /dev/null')
```

Comment:
You don't need root privilege to run Docker containers, if the user belongs to `docker` group.